### ISSUE TITLE
HTTP/1.1 400 Bad Request

### ISSUE THREAD

**Issue by gijsschumacher** [karthik/rDrop#16]
Dear rDrop developers, after some effort I managed to make rDrop work using paleo13's suggestion in this thread: https://github.com/karthik/rDrop/issues/4. Dropbox_save works using the 'curl=h' addition at the end. But when I try

```
package <- unserialize(dropbox_get(dropbox_credentials, file='parlvote.rda', curl=h))
share <- dropbox_share(dropbox_credentials, 'parlvote.rda', curl=h) 
```

I get this error

Error: Bad Request
< HTTP/1.1 400 Bad Request
< Server: nginx

What can I do to fix this?

Thanks!

**Comment by gijsschumacher** [comment]
Update. I managed to circumvent this error by using another way to get dropbox_auth to work (also in the thread I mentioned). But now I run into another problem, if I do 

```
a <- 1:4
b <- 5:8
c <- data.frame(a,b)
dropbox_save(dropbox_credentials, c, file='c')
package <- unserialize(dropbox_get(dropbox_credentials, file='c.csv'))
share <- dropbox_share(dropbox_credentials, 'c.csv')
```
Package and share both return this error:

Error in data.frame(rev = "260eb05338c6f", thumb_exists = FALSE, path = "/c.rda",  : 
  arguments imply differing number of rows: 1, 0

I have tried several different files and file types.
